auth/jwt_auth.go

```diff
@@ -8,10 +8,14 @@ import (
 	jwt "github.com/dgrijalva/jwt-go"
 )
 
-const privateKeyPath = "/run/secrets/derek-private-key"
+const secretKeyPath = "/run/secrets/derek-private-key"
 
 // GetSignedJwtToken get a tokens signed with private key
-func GetSignedJwtToken(appID string) (string, error) {
+func GetSignedJwtToken(appID string, privateKeyPath string) (string, error) {
+
+	if privateKeyPath == "" {
```

Comment:
I'm not keen on this logic being encapsulated in this function.. can we throw an error in this scenario?